lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -83,9 +74,14 @@ class FirstMeaningfulPaint extends Audit {
     });
   }
 
-  static calculateScore(data) {
-    // First meaningful paint is the last timestamp observed from the candidates
-    const firstMeaningfulPaint = (data.fMP.ts - data.navStart.ts) / 1000;
+  static calculateScore(evts) {
+    const firstMeaningfulPaint = (evts.firstMeaningfulPaint.ts - evts.navigationStart.ts) / 1000;
+    const firstContentfulPaint = (evts.firstContentfulPaint.ts - evts.navigationStart.ts) / 1000;
+
+    const timings = {};
+    timings.navStart = evts.navigationStart.ts / 1000;
```

Comment:
this is fair. Originally I was thinking of something like:

```js
{ 
  timestamps: {
    navStart: 8923942942.122,
    fCP: 8934942942.246,
    FMP: 8926942942.572
  },
  timings: {
    navStart: 0,
    fCP: 1201.33,
    FMP: 1642.55
  }
}
```

Happy to update it to this format.  And in fact, there are downstream consumers of the timestamps, so I'm happy to keep those around.